cloudferrylib/scheduler/scenario.py

```diff
@@ -57,8 +61,11 @@ def init_tasks(self, init):
     def load_scenario(self, path_scenario=None):
         if path_scenario is None:
             path_scenario = self.path_scenario
+        if not os.path.isfile(path_scenario):
+            raise NotFoundScenarioFileError(path_scenario)
```

Comment:
This check is repeated twice, I'd suggest that you isolate load of scenario file and raise this error only if file cannot be opened for read.